packages/core/src/index.ts

```diff
@@ -1,5 +1,5 @@
-export { createRouter, router } from './router'
-export type { Router, HybridRequestOptions, NavigationResponse, HybridPayload, ResolveComponent, Method, Progress } from './router'
+export { createRouter, router } from './router/router'
```

Comment:
This fixed an issue I was having where the tests would fail on the first run after installing the dependencies.

Reply:
That's interesting, do you know why this specifically caused the issue?

Reply:
Not sure. I had found some Google results about needing to `import * as x` but that only applies to default exports, could be a phantom fix.